ros2doctor/ros2doctor/api/format.py

```diff
@@ -12,9 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import sys
 
-def print_term(k, v):
-    """Print term only if it exists."""
-    if v:
-        # TODO(clairewang): 20 padding needs to be dynamically set
-        print('{:20}: {}'.format(k, v))
+
+def format_print(report):
+    """Format print report content."""
+    if report is None:
+        sys.stderr.write('WARNING: No report found. Skip print...\n')
```

Comment:
When can `format_print(None)` be called? It seems like that should raise an exception instead of printing a warning.

Reply:
In this case, if `ifcfg` is not imported successfully, the entire network module is skipped and report is None. I guess I should have added a note saying it is a temp fix. I will add the exception after `ifcfg` debian package becomes available.